scraper-tools/src/context.js

```diff
@@ -60,9 +60,11 @@ class Context {
     }
 
     async saveSnapshot() {
+        const { response } = this;
         return browserTools.saveSnapshot({
             page: this.page,
-            $: this.$,
+            body: this.body,
+            contentType: response && response.headers ? response.headers['content-type'] : null,
```

Comment:
Will `saveSnapshot` and therefore `Apify.setValue` be able to handle the charset in `contentType`?

Reply:
It should, as I set up right content type to value in kvs.